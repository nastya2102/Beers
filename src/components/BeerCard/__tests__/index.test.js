import React from "react";
import { render } from "@testing-library/react";
import BeerCard from "../index";

describe("BeerCard", () => {
  test("should render BeerCard component", () => {
    const { container } = render(
      <BeerCard
        name={true}
        image_url={"//"}
        tagline={<span>123</span>}
        abv={<span>457</span>}
        handleClickBeer={() => {}}
        id={1}
      />
    );

    expect(container).toBeTruthy();
  });

  test("should match snapshot BeerCard component", () => {
    const { container } = render(
      <BeerCard
        name={true}
        image_url={"//"}
        tagline={<span>123</span>}
        abv={<span>457</span>}
        handleClickBeer={() => {}}
        id={1}
      />
    );

    expect(container).toMatchSnapshot();
  });
});
