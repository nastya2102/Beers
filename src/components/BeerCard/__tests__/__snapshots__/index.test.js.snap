// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BeerCard should match snapshot BeerCard component 1`] = `
<div>
  <div
    class="sc-dlfnbm dfiElR"
  >
    <div
      class="sc-hKgILt cfsMEj"
    >
      <span>
        457
      </span>
      %
    </div>
    <div
      class="sc-gsTCUz fwkMwk"
    />
    <div
      class="sc-bdfBwQ tXWSF"
    />
    <div
      class="sc-eCssSg jgjKLH"
    >
      <span>
        123
      </span>
    </div>
  </div>
</div>
`;
